cpp/es1.2/medianaEdoardo.cpp

```diff
@@ -0,0 +1,54 @@
+#include<iostream>
+#include<vector>
+
+using namespace std;
+
+vector<double> bubleSort(vector<double> n){//algoritmo di sorting
```

Comment:
- You should avoid copying the whole vector like this: in this case you should pass the argument by reference and modify it, i.e., the new signature could be `void bubbleSort(vector<double>& v)`.
- The variable name `n` is a bad choice for a vector. Can you find a better name (e.g, `vec`, `v`, `numbers`...)?